psutil/_psbsd.py

```diff
@@ -432,6 +432,22 @@ def sensors_battery():
             secsleft = minsleft * 60
         return _common.sbattery(percent, secsleft, power_plugged)
 
+    def sensors_temperatures():
+        """Return systemp temperatures"""
+        ret = dict()
+        ret["coretemp"] = list()
+        num_cpus = cpu_count_logical()
+        try:
+            for cpu in range(num_cpus):
+                current, tjmax = cext.sensors_temperatures(cpu)
```

Comment:
I think it's better to catch `NotImplementedError` here instead of against the whole `for` loop. That way if there are temperatures for certain CPUs only (instead of all of them) we can at least return a partial result.

Also `tmax` looks like a better name than `tjmax`.